packages/visx-demo/src/components/Gallery/DelaunayTriangulationTile.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Delaunay, {
+  DelaunayTriangulationProps,
+} from '../../sandboxes/visx-delaunay-triangulation/Example';
+import GalleryTile from '../GalleryTile';
+
+export { default as packageJson } from '../../sandboxes/visx-delaunay-triangulation/package.json';
+
+const tileStyles = {
+  background: 'black',
+  borderRadius: 14,
+  boxShadow: 'rgba(0, 0, 0, 0.1) 0px 1px 6px',
+};
+const detailsStyles = { background: 'white', color: '#5B247A', borderRadius: '0 0 14px 14px' };
+
+export default function DelaunayTriangulationTile() {
```

Comment:
can you add a screenshot of this?

Reply:
Here is a screenshot of the tiles in the `@visx/delaunay` packages. The Voronoi tile is the same as before, and the Delaunay Triangulation tile is new.

<img width="814" alt="Screenshot 2023-07-05 at 10 08 41 PM" src="https://github.com/airbnb/visx/assets/50504641/497b2a10-97ab-48db-b196-b271b4424bf6">

Reply:
love it! ✨ 😍